lib/id3c/labelmaker.py

```diff
@@ -378,6 +378,12 @@ class CollectionsAirsLayout(LabelLayout):
     copies_per_barcode = 2
     reference = "fredhutch.org"
 
+class CollectionsCascadiaTinySwabsHome(SamplesLayout):
```

Comment:
Inheriting from SamplesLayout may not be the best solution here. Looking at the code, it may be better to add a `LCRY2380DuplicateLayout` class and then have `SamplesLayout` and `CollectionsCascadiaTinySwabsHome` inherit from that. This way, we can avoid duplication of the `blanks_before` function that this new collection inherits from `SamplesLayout` but not have this class inherit from another layout that is in use. Didn't add that at first since it is a bit more additional code, but is a simple change if whoever reviews this agrees it is a better solution.

Reply:
Hey @bencap, I agree that is a good idea, especially with the direction that a lot of studies may be going towards (tiny duplicates).